code/game/machinery/computer/depot.dm

```diff
@@ -84,17 +72,54 @@
 /obj/machinery/computer/syndicate_depot/proc/disable_special_functions()
 	return
 
-/obj/machinery/computer/syndicate_depot/Topic(href, href_list)
+/obj/machinery/computer/syndicate_depot/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = TRUE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "ComputerSimple",  name, window_width, window_height, master_ui, state)
+		ui.open()
+
+/obj/machinery/computer/syndicate_depot/ui_data(mob/user)
+	var/list/data = list()
+	data["rows"] = list()
+	if(security_lockout)
+		data["rows"].Add(list(list(
```

Comment:
```suggestion
		data["rows"] += list(list(
```
These warning can be suppressed by doing this. 

Explanation of warning: `data["rows"]` is not declared as a static type here, meaning that if `data["rows"]` was *not* equal to a list when the code was running, runtime errors would occur. The *proper* way to fix this is to cast to a list each time
```dm
var/list/rows = data["rows"]
rows.Add(whatever)
```
But that is a lot of extra lines, so ill allow `data["rows"] +=`